slices/erlang-tools.yaml

```diff
@@ -0,0 +1,18 @@
+package: erlang-tools
+
+essential:
+  - erlang-tools_copyright
+
+slices:
+  modules:
+    essential:
+      - erlang-base_bins
+      - erlang-runtime-tools_modules
+    contents:
+      /usr/lib/erlang/lib/tools-*/ebin/**:
+      /usr/lib/erlang/lib/tools-*/emacs/**:
```

Comment:
Is this support for emacs editor? If so, probably we should keep that in a separate slice.

Reply:
Indeed, those are emacs plugins. Moving them to a separate slice.